docs/DeveloperGuide.md

```diff
@@ -447,7 +447,7 @@ This feature has been enhanced based on the initial implementation of the `help`
 #### How is this feature implemented?
 
 The proposed `help` mechanism is facilitated by `HelpWindow`. It is supported by `HelpWindowUtil` where it will help the user open the User Guide on their desktop. 
-We used the existing implementation, and we modified it that the user can view the full user guide on their personal desktop with the click of a button. 
+We used the existing implementation, and we modified it that the user can view the full user guide on their candidateal desktop with the click of a button. 
```

Comment:
This might be a typo. 🤔

Reply:
Thanks @LeeEnEn  :) It must have been during the replacement from `person` --> `candidate`, but missed out on checking for `personal`! Have edited this!